src/types/index.d.ts

```diff
@@ -6,9 +6,14 @@ export interface MergeLink {
   }) => void;
   openLink: (config: UseMergeLinkProps) => void;
 }
+
+export interface TenantConfig {
+  apiBaseUrlOverride?: string
+}
 export interface UseMergeLinkProps {
   linkToken: string;
   demoCategories: Array<string>;
+  tenantConfig?: TenantConfig
   onExit: () => void;
   onLoad: () => void;
```

Comment:
Unrelated but needed before we publish an update to the NPM package: can you delete `onLoad` and `demoCategories`?